src/main/java/in/org/projecteka/hdaf/link/link/FetchUserId.java

```diff
@@ -0,0 +1,10 @@
+package in.org.projecteka.hdaf.link.link;
+
+import java.util.Base64;
+
+public class FetchUserId {
```

Comment:
FetchUserId sounds like a verb. Class names should be a noun. It can be named something like TokenReader. And also, it can be made static.